pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -240,6 +240,18 @@ func getPhase2Binding(vmi *v1.VirtualMachineInstance, iface *v1.Interface, netwo
 	if iface.Slirp != nil {
 		return &SlirpPodInterface{vmi: vmi, iface: iface, domain: domain}, nil
 	}
+	if iface.Macvtap != nil {
+		vif := &VIF{Name: podInterfaceName}
```

Comment:
AFAIU, `vif` is used for dhcp server. Here in macvtap we don't need dhcp server. So why `vif` is still required?